### DISCUSSION TITLE
TopsApp.py: No flattening requested?

### DISCUSSION THREAD

**Discussion by riyasmjgeo** [isce-framework/isce2#508] [Q&A] [answered]
Hello all,
I was generating an interferogram from Sentinel-1 IW SLC using topsApp.py application in ISCE2. In the log, I noticed a warning stating that 'No flattering requested'. Does this mean that the flat-earth component is not removed, and it needs to be specified explicitly in the XML?  The log message is pasted below, and the entire log is attached for reference. 
`2022-06-06 16:42:04,736 - isce.stdproc.resamp_slc - WARNING - The variable INPUT_LINES has been set to the default value 1506 which is the number of lines in the slc image.`
`2022-06-06 16:42:04,737 - isce.stdproc.resamp_slc - WARNING - The variable INPUT_WIDTH has been set to the default value 23818 which is the width of the slc image.`
`2022-06-06 16:42:04,737 - isce.stdproc.resamp_slc - WARNING - No flattening requested`
`2022-06-06 16:42:16,796 - isce.stdproc.resamp_slc - WARNING - The variable INPUT_LINES has been set to the default value 1506 which is the number of lines in the slc image.`
`2022-06-06 16:42:16,796 - isce.stdproc.resamp_slc - WARNING - The variable INPUT_WIDTH has been set to the default value 23818 which is the width of the slc image.`
`2022-06-06 16:42:16,796 - isce.stdproc.resamp_slc - WARNING - No flattening requested`
[isce.log](https://github.com/isce-framework/isce2/files/8945626/isce.log)

**Comment by fdelgadodelapuente** [comment, marked as answer]
if flat-earth fringes were not removed, your interferogram would have a huge ramp of parallel fringes. They are absent in your interferogram so ISCE corrects them.
